integration/framework/cli/cmd_base_custom_result.go

```diff
@@ -41,3 +45,31 @@ func checkRegex(r *regexp.Regexp, s string) assert.BoolOrComparison {
 	}
 	return true
 }
+
+func logs(result icmd.Result, args ...string) assert.BoolOrComparison {
+	var logEntriesCount int
+	if result.Stdout() != "" {
+		lines := strings.Split(result.Stdout(), "\n")
+		for _, l := range lines {
+			if len(l) == 0 {
+				continue
+			}
+			var x map[string]interface{}
+			if err := json.Unmarshal([]byte(l), &x); err != nil {
+				return err
+			}
+			logEntriesCount++
+		}
+	}
+
+	if len(args) > 0 {
+		expectedLogEntries, err := strconv.Atoi(args[0])
+		if err != nil {
+			return err
+		}
+		if logEntriesCount != expectedLogEntries {
+			return log.NewErrorf("unexpected number of log entries: %d", logEntriesCount)
+		}
+	}
+	return nil
```

Comment:
just for consistency with the regex function
```suggestion
	return true
```